### ISSUE TITLE
Change Azure Pipelines yaml to separate build of booter and CLR

### ISSUE THREAD

**Issue by josesimoes** [nanoframework/Home#357]
No need to keep building and releasing booter versions when nothing has changed. Currently nanoBooter is always build along with the nanoCLR which is a waste of time and computing resource. Not to mention that is misleading because you actually don't know if there were any changes or not.

**Comment by networkfusion** [comment]
We no longer use appveyor. Is this still applicable?

**Comment by Ellerbach** [comment]
@josesimoes is this still needed? I feel the build system has drastically evolved since 2019 :-)

**Comment by josesimoes** [comment]
Yes. Despite all that evolution having separate build for the booter still makes sense.
